src/components/WwwFrame/PromotionalBanner/Banners/DepositIncentiveBanner.vue

```diff
@@ -55,5 +77,29 @@ export default {
 			};
 		}
 	},
+	methods: {
+		activateBasketWatchQuery() {
```

Comment:
The apollo plugin does this watch automatically, so this isn't needed. Any time a different query gets a new result for a field that is used in the `amountToLendQuery` the `result` function above will be called with the updated value.